src/socialMediaActivity.js

```diff
@@ -733,7 +685,11 @@ function verifyFacebookComment({requestDetails = null}) {
 
     return {};
 }
+
 /**
+ * Determine the audience of a Facebook post or reshare.
```

Comment:
Here and throughout, when parsing platform-specific values, please make sure to document what the source data is and what assumptions you're making in parsing.